.travis.yml

```diff
@@ -83,6 +83,7 @@ script:
 
 before_deploy:
   - openssl aes-256-cbc -K $encrypted_945ec88d7d8f_key -iv $encrypted_945ec88d7d8f_iv -in .deploy/id_rsa.enc -out .deploy/id_rsa -d
+  - ls -ll .deploy/
```

Comment:
There seem to be an issue with deploying from `master` branch, this line was added for debugging purposes.